public/modules/custom/helfi_kymp_district_project_search/js/src/components/form/Text.tsx

```diff
@@ -48,19 +49,23 @@ export const Text = ({
     }
   }, [componentId, initialize, initialValue, loading, setQuery]);
 
-  const title: string = searchState[componentId]?.value?.[0]?.value;
+  useEffect(() => {
+    const newValue = !searchState?.[componentId]?.value ? '' : searchState[componentId]?.value?.[0]?.value;
```

Comment:
Is there ever a chance that value isn't an array?

Reply:
shouldn't be 🤞